index.js

```diff
@@ -13,7 +13,7 @@ const logoutRouter = require('./routes/logout');
 const riderRouter = require('./routes/rider');
 const staffRouter = require('./routes/staff');
 const managerRouter = require('./routes/manager');
-const signupBusinessRouter = require('./routes/signupBusiness')
+const signupBusinessRouter = require('./routes/signupBusiness');
```

Comment:
Shouldnt have a router for jus 1 function. Can put inside manager router

Reply:
This is out for scope of this pr. will make another one for this